LDAR_Sim/src/simulation/end_to_end_creation_simulation_manager.py

```diff
@@ -0,0 +1,124 @@
+import os
+import shutil
+from pathlib import Path
+from typing import Any, override
+
+import yaml
+from constants import param_default_const as pdc
+from constants.output_messages import RuntimeMessages as rm
+from file_processing.input_processing.input_manager import InputManager
+from file_processing.output_processing.summary_output_helpers import get_non_baseline_prog_names
+from initialization.args import files_from_path, get_abs_path
+from simulation.simulation_helpers import remove_non_preseed_files
+from simulation.simulation_manager import SimulationManager
+
+
+class EndToEndCreationSimulationManager(SimulationManager):
+    def __init__(
+        self,
+        input_manager: InputManager,
+        test_creator_dir: Path,
+        root_dir: Path,
+        script_arguments: list[str],
+    ):
+        self.GLOBAL_PARAMS_TO_REP: "dict[str, Any]" = {
+            pdc.Sim_Setting_Params.SIMS: 2,
+            pdc.Sim_Setting_Params.PRESEED: True,
+            pdc.Sim_Setting_Params.INPUT: "./inputs",
+            pdc.Sim_Setting_Params.OUTPUT: "./outputs",
+        }
+        self.GLOBAL_OUTPUT_PARAMS: "dict[str, Any]" = {
+            pdc.Output_Params.PROGRAM_VISUALIZATIONS: {
+                pdc.Output_Params.SINGLE_PROGRAM_TIMESERIES: False
+            }
+        }
+        self.test_creator_dir: Path = test_creator_dir
+        self.root_dir: Path = root_dir
+        self.parse_script_arguments(script_arguments)
+        self.clean_test_creator_dir()
+        self.retrieve_test_parameters()
+        self.fix_simulation_settings()
+        parameter_filenames: list[str] = files_from_path(self.params_dir)
+        super().__init__(input_manager=input_manager, parameter_filenames=parameter_filenames)
+
+    def parse_script_arguments(self, script_arguments: list[str]) -> None:
+        self.original_input_folder: str = script_arguments[1]
+        self.original_param_folder: str = script_arguments[2]
+        self.test_name: str = script_arguments[3]
+        self.sim_settings_file: str = script_arguments[4]
+
+    def clean_test_creator_dir(self) -> None:
+        for item in os.listdir(self.test_creator_dir):
+            item_path = os.path.join(self.test_creator_dir, item)
+            if item != ".gitignore":
+                if os.path.isfile(item_path):
+                    os.remove(item_path)
+                elif os.path.isdir(item_path):
+                    shutil.rmtree(item_path)
+
+    def retrieve_test_parameters(self) -> None:
+        self.original_inputs_dir: Path = self.root_dir / self.original_input_folder
+        self.original_params_dir: Path = self.root_dir / self.original_param_folder
+        self.params_dir: Path = self.test_creator_dir / "params"
```

Comment:
This I know is from the original, but do we have a constant that we can replace the "params" with?